README.md

```diff
@@ -714,6 +714,9 @@ Arguments:
 ## License
 visdom is Creative Commons Attribution-NonCommercial 4.0 International Public licensed, as found in the LICENSE file.
 
+## Note on Latex Support
+Now visdom support Latex. Check the examples/demo.py.
+
```

Comment:
Generally we use the demo to showcase possible functionality in visdom, so your addition to the demo is great and this note isn't necessary.

Reply:
Ok I will remove the README.md changed from the Pull Request.